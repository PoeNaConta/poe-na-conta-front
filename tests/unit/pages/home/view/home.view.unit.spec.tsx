import React from 'react';
import Home from '@pages/home/view/home.view';

import { describe, expect, it } from 'vitest';
import { render } from '@testing-library/react';

describe('Home', () => {
  it('should render home', () => {
    const { container } = render(<Home />);

    expect(container).toMatchSnapshot();
  });
});
