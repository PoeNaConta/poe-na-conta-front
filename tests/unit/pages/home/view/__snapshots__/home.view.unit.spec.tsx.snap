// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Home > should render home 1`] = `
<div>
  <p>
    Home Page
  </p>
</div>
`;
